lib/core/parse_config.cpp

```diff
@@ -338,3 +371,21 @@ bool parse_config_result::should_error_at_exit() const
   return action == action_t::exit_with_error;
 }
 
+std::ostream& metashell::operator<<(
+  std::ostream& out_,
+  parse_config_result::action_t a_
+)
+{
+  switch (a_)
+  {
+  case parse_config_result::action_t::run_shell:
+    return out_ << "run_shell";
+  case parse_config_result::action_t::exit_with_error:
+    return out_ << "exit_with_error";
+  case parse_config_result::action_t::exit_without_error:
+    return out_ << "exit_without_error";
+  default:
+    assert(!"Invalid action");
```

Comment:
I've added std::abort() there. assert(false) means that if the execution gets there, the program is in an invalid state ("this should never happen" :))